app.js

```diff
@@ -10,9 +10,7 @@ const dbConfig = require('./database/db');
 var cookieparser =require('cookie-parser');
 const csurf = require('csurf');
 var multer = require('multer');
-var upload = multer({
-  limits: { fileSize: 1024*1024 }
-});
+var upload = multer();
```

Comment:
#### Automated PR Comment From coverity
<!-- 6c2fd2fb50619d4e390c40961c78695d -->
<!-- Comment managed by Git Commenter, do not modify! -->
<!-- Comment managed by coverity-github-pr-commenter, do not modify!
6c2fd2fb50619d4e390c40961c78695d
-->
# Coverity Issue - Unrestricted number of uploaded file/non-file fields
_High CWE-400_
The `multer` module does not restrict the number of file/non-file fields in a single request. Thus, a denial-of-service attack is possible, because an attacker may submit a very large number of file/non-file fields in one request.
## How to fix
Limit the number of file fields in a request by setting the `files` property and the number of non-file fields by setting the `fields` property in the `limits` object of the `multer` options.